pkg/grub/Dockerfile

```diff
@@ -18,7 +18,7 @@ RUN apk add \
 RUN ln -s python3 /usr/bin/python
 
 ENV GRUB_MODULES="part_gpt fat ext2 iso9660 gzio linux acpi normal cpio crypto disk boot crc64 gpt \
-search_disk_uuid verify xzio xfs video multiboot2 gfxterm efi_gop efi_uga gptprio chain probe reboot "
+search_disk_uuid verify xzio xfs video gfxterm efi_gop gptprio chain probe reboot "
 ENV GRUB_COMMIT=6782f6d431d22b4e9ab14e94d263795c7991e160
```

Comment:
By doing this you are breaking x86. multiboot2 is required to boot xen on EFI. As you're not using it, can you just disable grub on ARM? Also because bootloading on ARM will be highly platform specific I have come to understand.